be/src/runtime/current_thread.cpp

```diff
@@ -31,7 +31,9 @@ CurrentThread::~CurrentThread() {
 
 starrocks::MemTracker* CurrentThread::mem_tracker() {
     if (UNLIKELY(tls_mem_tracker == nullptr)) {
-        tls_mem_tracker = ExecEnv::GetInstance()->process_mem_tracker();
+        if (ExecEnv::is_init()) {
+            tls_mem_tracker = ExecEnv::GetInstance()->process_mem_tracker();
+        }
```

Comment:
here allow to return null for `tls_mem_tracker`?

Reply:
yes, outside has null checks